R/app_ui.R

```diff
@@ -5,7 +5,8 @@
 #' @import shiny
 #' @noRd
 app_ui <- function(request) {
-
+  i18n <<- shiny.i18n::Translator$new(
+    translation_json_path = system.file("translation.json", package = "peskas.timor.portal"))
```

Comment:
This should not be necessary as the translation object is created in start_fun(). The app should be run using `run_app(onStart = start_fun)`